getting_started/intel_debug/introduction.rst

```diff
@@ -104,6 +104,18 @@ installing their own firmware. Notable exceptions include Google
 Chromebooks and Up2/Up-Extreme boards, where the 'community key' is
 used.
 
+The Intel ME (Management Engine) is responsible for authentication of
+the firmware, be it signed by an Intel production key, a community key
+or an OEM key. If the Intel ME is disabled by an OEM, or disabled by
```

Comment:
..., be it signed by an Intel or OEM production key (consumer products) or by a community key (development systems and make boards)

Reply:
Chromebooks also use community key.